auxiliaries/file_writer.py

```diff
@@ -1,24 +1,24 @@
-import logging
-logger = logging.getLogger('main')
-
-
-def write_to_file(file_path, content=''):
-    if not content:
-        import traceback
-        content = ''.join(traceback.format_stack())
-
-    with open(file_path, 'w') as f:
-        f.write(content)
-
-    logger.debug(f'{file_path} was generated.')
-
-
-if __name__ == '__main__':
-    # This block will be executed only when you run it as your main program.
-    # If this module is being imported from another script, this block won't be executed, however the function will be available...
-    import argparse
-    parser = argparse.ArgumentParser()
-    parser.add_argument('file_path', help='A path to a file to write to')
-    parser.add_argument('--content', help='The content that will be written to the file', default='')
-    args = parser.parse_args()
-    write_to_file(args.file_path, args.content)
+import logging

```

Comment:
There many changes due to end of line change (\r\n instead of just \n).
We would like to preserve Linux conventions (just \n).
This also makes it hard to review to PR as there too many non-relevant changes.
Please fix all the files via replace all